static/app.webmanifest

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Zulip (Web)",
+    "short_name": "Zulip",
+    "description": "Organized chat for distributed teams.",
+    "start_url": "/",
+    "display": "standalone",
```

Comment:
Can you talk about how you tested this? It might be annoying to test without the mobile apps, so if it makes sense to test-deploy to chat.zulip.org, I can do that.

Reply:
I have tested this using Chrome for now. It is on my todo, to test it on other devices today. But according to https://web.dev/articles/install-criteria this should work; although these doc is mostly aimed towards chrome.

Reply:
A test-deployment on `chat.zulip.org` would be really helpful with testing the manifest.

Reply:
There would also help with testing while fulfilling the requirement: "For a PWA to be installable it must be served using the https protocol, or from a local development environment using localhost or 127.0.0.1"

Reply:
Yeah we can crowdsource testing once it's on chat.zulip.org.